server/src/test/java/org/elasticsearch/common/time/DateFormattersTests.java

```diff
@@ -127,20 +127,20 @@ public void testTimeZones() {
     public void testEqualsAndHashcode() {
         assertThat(DateFormatters.forPattern("strict_date_optional_time"),
```

Comment:
this change is not needed either, as this does not matter which format it is within this test